File: api/app/Model/Group.php
<?php

namespace App\Model;

class Group extends Model
{
    protected $connection = 'default';
    protected $table = 'group';
    protected $fields = [
        'id' => 'int',
        'name' => 'string',
        'status' => 'int',
        'remark' => 'string',
        'created_time' => 'string',
        'updated_time' => 'string',
    ];

    const ADMIN_ID = 1;

    public function isAdmin(array $gids): bool
    {
        return \in_array(self::ADMIN_ID, $gids);
    }

    protected function _parseList(array $list): array
    {
        foreach ($list as $k => $v) {
            $list[$k] = $this->_parse($v);
        }
        return $list;
    }

    protected function _parse(array $data): array
    {
        return $data;
    }

}

File: api-gin/internal/models/group.go
package models

import "time"

type GroupModel struct {
	Conn  string
	Table string
}

var GroupMdl = &GroupModel{"default", "group"}

type GroupRow struct {
	ID          int64     `db:"id"`
	Name        string    `db:"name"`   //  群组名称
	Status      int64     `db:"status"` //  状态（1-正常，0-禁用）
	Remark      string    `db:"remark"`
	CreatedTime time.Time `db:"created_time"`
	UpdatedTime time.Time `db:"updated_time"`
}
